test-case/check-keyword-detection.sh

```diff
@@ -112,7 +112,8 @@ _update_blob(){
     [ $preamble_time -ge $history_depth ] || {
         die "Warning: invalid arguments, preamble_time must be greater than or equal to history_depth"
     }
-    awk -F "," '{if ( $4 == '"$def_pt"' && $7 == '"$def_hd"' ) $4='"$preamble_time"'; $7='"$history_depth"'}1' "$def_blob" > "$new_blob"
+    awk 'BEGIN{FS=OFS=","} {if ( $4 == '"$def_pt"' && $7 == '"$def_hd"' ) $4='"$preamble_time"'; \
+        $7='"$history_depth"'}1' "$def_blob" > "$new_blob"
```

Comment:
To reduce the amount of quotation needed and make the code slightly easier to read:
```shell
awk -F, -v OFS=, '{ if ...
```

Reply:
Updated.